.github/workflows/update-staging-stack.yml

```diff
@@ -57,6 +57,11 @@ jobs:
         commit_message: 'chore: updated staging stack ${{ env.FILE_NAME }}'
 
     - name: Summary step
+      env:
+        # REPOSITORY: repository that will update current version.
+        REPOSITORY: ${{ github.event.client_payload.repository }}
+        # TAG: tag pushed by the repository.
+        TAG: ${{ github.event.client_payload.tag }}
       run: |
         LATEST_FILE_LINK="https://github.com/${{ github.repository }}/blob/main/staging-versions/${{ env.FILE_NAME }}"
-        echo "### :postbox: Updated $REPOSITORY to `$TAG`\n\n[Go to generated staging version file]($LATEST_FILE_LINK)" >> $GITHUB_STEP_SUMMARY
+        echo "### :postbox: Updated $REPOSITORY to `$TAG`  \n[Go to generated staging version file]($LATEST_FILE_LINK)" >> $GITHUB_STEP_SUMMARY
```

Comment:
Has the line return been replaced by 2 spaces on purpose?

Reply:
Yes, the return did not seem to work, so i resorted to the 2 spaces strategy to create a line return.